cloudflare/resource_cloudflare_spectrum_app.go

```diff
@@ -0,0 +1,313 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/pkg/errors"
+)
+
+const (
```

Comment:
Do we gain anything from using the two `const` here? It's only used in a single place so I can't imagine it would get much reuse at the moment. Perhaps we wait and see if other resources need it and then extract?

Reply:
Looking further down, these aren't the only available options so perhaps they can go entirely.

Reply:
sure i'll remove the constants. I normally use constants for known string values as it makes it easier to reference them from code and tests. But happy to take your stear